### ISSUE TITLE
[Feature] - Windows support

### ISSUE THREAD

**Issue by alefesta** [k3ai/k3ai#11]
We should aim to support Windows, more specifically WSL2.

Current challenges:
- Some K8s flavors require systemd (i.e.: K3s) so we should use an alternative like K3d
- Expose ports could be challenging
- How do we discriminate between the various OS in the plugins?

**Comment by burntcarrot** [comment]
I use `kind` for local cluster development, but I'd like more priority for `k3d` than `k3s`. `k3s` doesn't work well with WSL2.

**Comment by alefesta** [comment]
Yep, I was thinking of actually adding k3d as an alternative to the plugins... That's a good point would you mind opening an issue for K3d so I can track it?

**Comment by burntcarrot** [comment]
Opened #25 in favor of this issue.
